extensions/ql-vscode/package-lock.json

```diff
@@ -1,8 +1,12255 @@
 {
   "name": "vscode-codeql",
   "version": "1.4.3",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
```

Comment:
Does this mean we have to use Node 14 / NPM 7? If so worth updating the contributor instructions.

Reply:
this lock file format is backwards compatible with npm v5 and v6. And theoretically compatible with v12. Looking at the readme now, it shows some outdated dependencies. It will _probably_ work with them, but we're not testing on them any more, so I'll update it.